src/v/debug_bundle/debug_bundle_service.cc

```diff
@@ -70,6 +72,12 @@ std::filesystem::path form_debug_bundle_file_path(
   const std::filesystem::path& base_path, job_id_t job_id) {
     return base_path / fmt::format("{}.zip", job_id);
 }
+
+bool is_valid_k8s_namespace(std::string_view ns) {
+    constexpr auto max_ns_length = 253;
```

Comment:
I think this might be 63.

Reply:
Last meeting we had, someone sad 253... I'll ask in the kubernetes channel

Reply:
> Last meeting we had, someone sad 253... I'll ask in the kubernetes channel

I said that, but I think I was wrong.

This is how I read it:
1. [As a result, all namespace names must be valid RFC 1123 DNS labels.](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/#namespaces-and-dns:~:text=As%20a%20result%2C%20all%20namespace%20names%20must%20be%20valid%20RFC%201123%20DNS%20labels)
2. [contain at most 63 characters](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-label-names:~:text=1123.%20This%20means%20the%20name%20must%3A-,contain%20at%20most%2063%20characters,-contain%20only%20lowercase%20alphanumeric%20characters%20or%20%27%2D%27)

Reply:
slack thread for ref: https://redpandadata.slack.com/archives/C01H6JRQX1S/p1727345850571989